client/webserver/site/src/js/markets.ts

```diff
@@ -1398,6 +1398,7 @@ export default class MarketsPage extends BasePage {
     app().log('book', 'handleBookRoute:', note)
     const mktBook = note.payload
     const market = this.market
+    if (!market?.dex) return
```

Comment:
This is part of an order book subscription that is initiated with `'loadmarket'`, which isn't called until after we've set `this.market`. I'm pretty sure will have `this.market` here. Same with `handleBookOrderRoute`, `handleUnbookOrderRoute`, `handleUpdateRemainingRoute`, `handleEpochOrderRoute`, `handleCandlesRoute`, and `handleCandleUpdateRoute`. 

As far as I can tell, the only places we could potentially get an update before `this.market` is set is `handleEpochNote`, `handlePriceUpdate`, and `handleBondUpdate`.

Reply:
Okay, I didn't trace all note handlers that are initiated **after** `this.market` is ready. Thanks.